sdks/go/container/build.gradle

```diff
@@ -53,6 +53,7 @@ docker {
           root: project.rootProject.hasProperty(["docker-repository-root"]) ?
                   project.rootProject["docker-repository-root"] :
                   project.docker_image_default_repo_root)
+  tags containerImageTags()
```

Comment:
Can you add comments that say this property only gets used by dockerTagPush? I think it will be confusing otherwise.

Reply:
Done